lib/mobility-core/src/Kernel/External/Payment/Interface/Types.hs

```diff
@@ -28,7 +28,7 @@ newtype PaymentServiceConfig = JuspayConfig Juspay.JuspayCfg
 
 data CreateOrderReq = CreateOrderReq
   { orderShortId :: Text, -- Should be Alphanumeric with character length less than 18.
-    amount :: Money,
+    amount :: HighPrecMoney,
```

Comment:
Can you check if session API can handle decimal difference?

Reply:
Changed back to money